pandas/core/generic.py

```diff
@@ -2184,6 +2184,12 @@ def _repr_data_resource_(self):
     automatically chosen depending on the file extension):
 
     >>> df1.to_excel('output1.xlsx', engine='xlsxwriter')  # doctest: +SKIP
+
+    To append data to an existing excel file:
+
+    >>> with pd.ExcelWriter('the_existing_file.xlsx', engine='openpyxl',
+    ...                     mode='a') as writer:
+    ...     df.to_excel(writer)
```

Comment:
This example feels disconnected with the rest of the docstring.

I'd move it to line 2181, and say something like `ExcelWriter can also be used to append to an existing Excel file`. And then use the same file `output.xlsx` and append a sheet `Sheet_name_3`. I think that will be much clearer.

You also need the `# doctest: +SKIP` as in the other examples.

Reply:
Yeah. You are right. That will be clearer. Will make the required change